templates/base_index.html

```diff
@@ -1382,8 +1382,15 @@ <h2>
           var takeovers = JSON.parse(xhr.responseText);
 
           // Get the selected takeovers based on the primary language
+          var currentUserCountry = "{{ current_user_country }}"
```

Comment:
Why "current"? Does it change during the immediate session?

Reply:
Well, I named it this because it is the current user provide by the `flask.request`, so it will change when another user visits this page. But I can rename it